doc/user/content/sql/select.md

```diff
@@ -1,35 +1,38 @@
 ---
 title: "SELECT"
-description: "`SELECT` reads data from your Kafka streams or materialized views."
+description: "`SELECT` reads from or queries your materialized views."
 menu:
   main:
     parent: 'sql'
 ---
 
-`SELECT` is used in a few ways within Materialize, letting you:
+`SELECT` is used in a few ways within Materialize. You can use it to:
 
-- Express a view you want to materialize. e.g. `CREATE MATERIALIZED VIEW some_view AS
-  SELECT...`
-- Read from materialized views or sources, e.g. `SELECT * FROM some_view;`
+- Query materialized views and materialized sources, e.g. `SELECT * FROM some_view;`
+- Describe a view you want to materialize. e.g. `CREATE MATERIALIZED VIEW some_view AS SELECT...`
 - Describe a view without materializing it, e.g. `CREATE VIEW some_nonmaterialized_view AS...`
 
 To better understand the distinction between these uses, you should check out
 our [architecture overview](../../overview/architecture).
 
 ## Conceptual framework
 
+The `SELECT` statement is the root of a SQL query, and is used both to interactively query data Materialize manages, and to bind SQL queries to named views which can be queried later.
+
+When used to interactively query data, your query must depend only on materialized sources and views. There are exceptions, discussed in more detail below. When used to construct views, this restriction is lifted.
+
 To perform reads, Materialize simply returns the result set of a dataflow.
 (Naturally, if the dataflow doesn't exist, it must be created.)
```

Comment:
I didn't mean to leave this, but I feel like a dufus sending in another commit just to scrub it. I thought it was a bit short for what is a pretty important distinction (reading v querying).